python/hawkey/goal-py.c

```diff
@@ -463,6 +463,27 @@ describe_problem(_GoalObject *self, PyObject *index_obj)
 }
 
 static PyObject *
+describe_problem_rules(_GoalObject *self, PyObject *index_obj)
+{
+    PyObject *list;
+
+    if (!PyInt_Check(index_obj)) {
+        PyErr_SetString(PyExc_TypeError, "An integer value expected.");
+        return NULL;
+    }
+    const char **plist = hy_goal_describe_problem_rules(self->goal,
+                                                       PyLong_AsLong(index_obj));
+    if (plist == NULL) {
+        PyErr_SetString(PyExc_ValueError, "Index out of range.");
+        return NULL;
+    }
+
+    list = strlist_to_pylist(plist);
+    g_free(plist);
```

Comment:
It's g_free() to make it consistent with efcc17842925a3c4867dbfa9c9b491580bdc8001.